apps/client/src/pages/loading/Loading.page.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactElement } from 'react';
+import LoaderIcon from 'pixelarticons/svg/loader.svg';
+
+export function Loading(): ReactElement {
+  return (
+    <div className='flex h-full'>
+      <div className='m-auto flex gap-2'>
+        <img src={LoaderIcon} alt='' width='200px' height='200px' />
```

Comment:
Why is the width and height set using inline attributes instead of tailwind classes?

Reply:
done c92a80e23f0e9581fb997bfae5e9e073687174ec